This fixes a few compiler and linker errors.

don't ask... please... just don't ask
blindcoder
--- ./win/Qt/qt_win.cpp	2003-06-08 20:56:11.000000000 +0200
+++ ./win/Qt/qt_win.cpp	2003-06-08 14:27:30.000000000 +0200
@@ -47,6 +47,8 @@ extern "C" {
 // enough, and we undefine NetHack macros which conflict with Qt
 // identifiers.
 
+#include <math.h>
+
 #define alloc hide_alloc // avoid treading on STL symbol
 #define lock hide_lock // avoid treading on STL symbol
 #ifdef _MSC_VER
--- ./src/Makefile 2003-06-08 20:56:11.000000000 +0200
+++ ./src/Makefile 2003-06-08 14:27:30.000000000 +0200
@@ -147,1 +147,1 @@
-	winmisc.o winstat.o wintext.o winval.o tile.o
+	winmisc.o winstat.o wintext.o winval.o
@@ -179,1 +179,1 @@
-WINQTOBJ = qt_win.o qt_clust.o qttableview.o tile.o
+WINQTOBJ = qt_win.o qt_clust.o qttableview.o
@@ -191,1 +191,1 @@
-	gnyesno.o gnworn.o tile.o
+	gnyesno.o gnworn.o
